### PR TITLE
Fix k8s_object to work with latest rules_docker (#130)

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

File: k8s/object.bzl [hunk 1]
```diff
@@ -65,7 +63,7 @@ def _impl(ctx):
     # to include as runfiles, so they are accessible to be pushed.
     for tag in ctx.attr.images:
       target = ctx.attr.images[tag]
-      image = _get_layers(ctx, image_target_dict[target], image_files_dict[target])
+      image = _get_layers(ctx, image_target_dict[target])
```
Comment (drigz) [inline k8s/object.bzl]: FYI, the signature changed again - now it's `(ctx, name, attr_target)` ([more here](https://github.com/bazelbuild/rules_k8s/issues/125#issuecomment-395416493)).

**Comment by fejta** [comment]
Matt, can you update and/or close?

**Comment by fejta** [comment]
Fixed by https://github.com/bazelbuild/rules_k8s/pull/150
/close

**Pull request closed without merging**